resources/hvac.rb

```diff
@@ -1900,7 +1996,12 @@ def self.disaggregate_fan_or_pump(model, fan_or_pump, htg_objects, clg_objects)
 
     clg_object_sensors = []
     clg_objects.each do |clg_object|
-      clg_object_sensor = OpenStudio::Model::EnergyManagementSystemSensor.new(model, "Cooling Coil Electric Energy")
+      if clg_object.is_a? OpenStudio::Model::EvaporativeCoolerDirectResearchSpecial
+        var = "Evaporative Cooler Electric Energy"
```

Comment:
Need assumption for recirculating water pump power, or this variable will always report 0, and fan power will not be correctly disaggregated.

Reply:
Used evap cooler water consumption to disaggregate.